app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntrySheet.kt

```diff
@@ -393,6 +393,18 @@ class BloodSugarEntrySheet : BottomSheetActivity(), BloodSugarEntryUi, RemoveBlo
     bloodSugarReadingUnitLabel.visibility = View.VISIBLE
   }
 
+  override fun setInputTypeForBloodSugarMmol() {
```

Comment:
Probably instead of creating 3 methods for switching the input types, we can have `decimalBloodSugarInputType` and `numberBloodSugarInputType` (you can think of better naming). That way we will have 2 methods that are not tied to specific blood sugar type or unit.

Reply:
Great observation. Changing this to have two methods.